### ISSUE TITLE
App crashed 

### ISSUE THREAD

**Issue by manish7924** [AndroidIDEOfficial/AndroidIDE#1305]
### What happened?

App has crashed

### What's the expected behavior?

App is crashed 

### What version of AndroidIDE you're using?

v2.5.2 (debug builds)

### Relevant log output

```shell
AndroidIDE crash report
Manufacturer: realme
Device: RMX3624
ABI: arm64-v8a
SDK version: 33
App version: v2.5.2-beta (252)

 Stacktrace: 
android.view.WindowManager$BadTokenException: Unable to add window -- token null is not valid; is your activity running?
	at android.view.ViewRootImpl.setView(ViewRootImpl.java:1304)
	at android.view.WindowManagerGlobal.addView(WindowManagerGlobal.java:411)
	at android.view.WindowManagerImpl.addView(WindowManagerImpl.java:221)
	at android.widget.PopupWindow.invokePopup(PopupWindow.java:1581)
	at android.widget.PopupWindow.showAtLocation(PopupWindow.java:1347)
	at android.widget.PopupWindow.showAtLocation(PopupWindow.java:1313)
	at io.github.rosemoe.sora.widget.base.EditorPopupWindow.applyWindowAttributes(Unknown Source:148)
	at com.itsaky.androidide.editor.ui.AbstractPopupWindow.show(Unknown Source:23)
	at com.itsaky.androidide.editor.ui.EditorActionsMenu.show(Unknown Source:215)
	at com.itsaky.androidide.editor.ui.EditorActionsMenu.displayWindow(Unknown Source:172)
	at com.itsaky.androidide.editor.ui.EditorActionsMenu$$ExternalSyntheticLambda1.run(Unknown Source:15)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:8061)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:703)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:923)
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by remo7777** [comment]
AndroidIDE crash report
Manufacturer: realme
Device: RMX1921
ABI: arm64-v8a
SDK version: 30
App version: v2.5.3-beta.internal.1+branch.dev.commit.42ac697.timestamp.20230909074201 (253)

 Stacktrace: 
android.os.RemoteException: Remote stack trace:
	at com.android.server.pm.PackageInstallerSession.markAsSealed(PackageInstallerSession.java:1405)
	at com.android.server.pm.PackageInstallerSession.commit(PackageInstallerSession.java:1161)
	at android.content.pm.IPackageInstallerSession$Stub.onTransact(IPackageInstallerSession.java:328)
	at android.os.Binder.execTransactInternal(Binder.java:1161)
	at android.os.Binder.execTransact(Binder.java:1130)

	at com.android.server.pm.PackageInstallerSession.markAsSealed(PackageInstallerSession.java:1405)
	at com.android.server.pm.PackageInstallerSession.commit(PackageInstallerSession.java:1161)
	at android.content.pm.IPackageInstallerSession$Stub.onTransact(IPackageInstallerSession.java:328)
	at android.os.Binder.execTransactInternal(Binder.java:1161)
	at android.os.Binder.execTransact(Binder.java:1130)
 Caused by: java.lang.SecurityException: Can't install packages while in secure FRP
	at android.os.Parcel.createExceptionOrNull(Parcel.java:2389)
	at android.os.Parcel.createException(Parcel.java:2373)
	at android.os.Parcel.readException(Parcel.java:2356)
	at android.os.Parcel.readException(Parcel.java:2298)
	at android.content.pm.IPackageInstallerSession$Stub$Proxy.commit(IPackageInstallerSession.java:657)
	at android.content.pm.PackageInstaller$Session.commit(PackageInstaller.java:1250)
	at com.itsaky.androidide.utils.ApkInstaller$installApk$2.invoke(ApkInstaller.kt:94)
	at com.itsaky.androidide.utils.ApkInstaller$installApk$2.invoke(ApkInstaller.kt:87)
	at com.itsaky.androidide.tasks.TaskExecutorKt.executeAsync$lambda$1(TaskExecutor.kt:94)
	at com.itsaky.androidide.tasks.TaskExecutorKt.$r8$lambda$gunyMTwaq5sezOICXKqDbc6KGCY(Unknown Source:0)
	at com.itsaky.androidide.tasks.TaskExecutorKt$$ExternalSyntheticLambda3.complete(Unknown Source:2)
	at com.itsaky.androidide.tasks.TaskExecutor$executeAsync$2.invoke$lambda$0(TaskExecutor.kt:45)
	at com.itsaky.androidide.tasks.TaskExecutor$executeAsync$2.$r8$lambda$-1toc0O_VsFrdzpjT4CZnZHglSY(Unknown Source:0)
	at com.itsaky.androidide.tasks.TaskExecutor$executeAsync$2$$ExternalSyntheticLambda0.run(Unknown Source:4)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:263)
	at android.app.ActivityThread.main(ActivityThread.java:8296)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:612)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1006)

**Comment by itsaky** [comment]
@remo7777 The device seems to be in Factory Reset Protection (FRP) mode which is activated when the device is hacked/stolen/modified.

We can't do anything about that.

**Comment by itsaky** [comment]
Fixed with 940c7b0aa22bc5ee47e8263015089003d1f72b8d.
